### PR TITLE
Refactor: `import-targets` to `import-artifacts`

### PR DESCRIPTION
**Pull request by kairoaraujo**
# Description
- Updates the user commands to show as `import-artifacts` instead `import-targets` following the changes in the API and documentation.

# Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

# Additional requirements
- [x] Tests have been added for the bug fix or new feature
- [x] Docs have been added for the bug fix or new feature

# Code of Conduct
By submitting this PR, you agree to follow our [Code of Conduct](CODE_OF_CONDUCT.rst).

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: docs/source/guide/index.rst [hunk 1]
```diff
@@ -711,13 +712,14 @@ sign (``sign``)
 
 
 .. rstuf-cli-admin-import-targets
```
Comment (MVrachev) [inline docs/source/guide/index.rst:714]:
```suggestion
```

File: docs/source/guide/index.rst [hunk 2]
```diff
@@ -47,13 +47,14 @@ It executes administrative commands to the Repository Service for TUF.
...
-    │ import-targets                                        Import targets to RSTUF from exported CSV file.                                                                                                                        │
-    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Options ────────────────────────────────────────────────────────────────────╮
+    │ --help  -h    Show this message and exit.                                    │
+    ╰──────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Commands ───────────────────────────────────────────────────────────────────╮
+    │ ceremony                    Start a new Metadata Ceremony.                   │
+    │ import-artifacts            Import artifacts to RSTUF from exported CSV file.│
```
Comment (MVrachev) [inline docs/source/guide/index.rst:55]:
```suggestion
    │ import-artifacts            Import artifacts to RSTUF from exported CSV file. │
```

**Review by MVrachev — COMMENTED** [review]
A few recommendations where to rename `targets` to artifacts: 
- https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/pyproject.toml#L49-L50
- https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/repository_service_tuf/cli/admin/ceremony.py#L216
- !!! BUG !!! https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/repository_service_tuf/helpers/api_client.py#L23 I am in favor of renaming the enum URL from `public_targets` to `public_artifacts` here
- If you rename `publish_targets` above you can consider doing here: https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/980495ad51ee0cc3a847c24d0ceb4de9eec1b1d6/repository_service_tuf/helpers/cli.py#L45 and here https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/repository_service_tuf/helpers/api_client.py#L161

File: docs/source/guide/index.rst (hunk 2 above)
Comment (MVrachev) [reply docs/source/guide/index.rst:55]: If you do that you need to add a space to the other rows as well.

File: docs/source/guide/index.rst (hunk 1 above)
Comment (kairoaraujo) [reply docs/source/guide/index.rst:714]: It is to not break the docs in the umbrella until we update the sphinx import links.

File: docs/source/guide/index.rst (hunk 2 above)
Comment (kairoaraujo) [reply docs/source/guide/index.rst:55]: I think we can keep it. It is just an example to the user in the docs., unless there are other changes to do.

**Comment by kairoaraujo** [comment]
> A few recommendations where to rename `targets` to artifacts:
> 
> * https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/pyproject.toml#L49-L50

Done
> * https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/repository_service_tuf/cli/admin/ceremony.py#L216

Done
> * !!! BUG !!! https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/repository_service_tuf/helpers/api_client.py#L23 I am in favor of 
renaming the enum URL from `public_targets` to `public_artifacts` here

> * If you rename `publish_targets` above you can consider doing here: https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/980495ad51ee0cc3a847c24d0ceb4de9eec1b1d6/repository_service_tuf/helpers/cli.py#L45
>    and here https://github.com/kairoaraujo/repository-service-tuf-cli/blob/4d92647e388a496fda5cce45abc5cf707850b200/repository_service_tuf/helpers/api_client.py#L161

I'm not renaming now. We can do it as another issue. This PR was intended to change the user views.

**Review by MVrachev — APPROVED** [review]
LGTM!

Added an issue for the rest of my comments: https://github.com/repository-service-tuf/repository-service-tuf-cli/issues/422

**Pull request merged**